packages/logs/src/domain/logger.spec.ts

```diff
@@ -94,6 +94,16 @@ describe('Logger', () => {
       })
     })
 
+    it('should be configurable to "console" and "http"', () => {
+      logger.setHandler(['console', 'http'])
```

Comment:
While using values directly is legitimate, we tend to prefer using the constant `HandlerType` internally instead